recipes/restinio/all/conanfile.py

```diff
@@ -0,0 +1,79 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class RestinioConan(ConanFile):
+    name = "restinio"
+    license = "BSD-3-CLAUSE"
+    homepage = "https://github.com/Stiffstream/restinio"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "RESTinio is a header-only C++14 library that gives you an embedded HTTP/Websocket server."
+    topics = ("http-server", "websockets", "rest", "tls-support")
+    exports_sources = ["CMakeLists.txt"]
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"use_boost": [True, False], "use_openssl": [True, False]}
+    default_options = {"use_boost": False, "use_openssl": False}
+    generators = "cmake"
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def configure(self):
+        tools.check_min_cppstd(self, "14")
```

Comment:
Currently you need to check if the cppstd setting is present before making this call

Reply:
In the doc it says that it deduce the default standard used by the compiler if no compiler.cppstd is defined : https://docs.conan.io/en/latest/reference/tools.html#tools-check-min-cppstd

I think it's enough, no ?